### DISCUSSION TITLE
Collections not adding items that I have

### DISCUSSION THREAD

**Discussion by edrock200** [meisnate12/Plex-Meta-Manager#130] [Q&A]
Hello,
First off, thanks for making this awesome utility. Definitely a huge convenience and time saver. I'm not sure if I'm doing something wrong, but I have a fairly simple setup. It has a plex server in the main config, and several trakt lists in a TV and Movies yml.

For some reason, some of the collections don't populate with items that I have that are in the trakt list. For example, if I make a "Disney TV Shows" collection tied to a trakt list with "Wandavision," Wandavision won't show up in the collection.

I'm not sure what to provide to help narrow the issue but here's some info if it helps:
I've tried this against two plex servers, one with Movie and TV Libraries on legacy plex scanners, and the other with both libraries updated to the latest scanner.

At one point with both, the agent was switched from the main scanner for TV Shows to "themoviedb", then subsequently reverted back after due to some issues in pulling metadata. I don't know if this has an impact.

I've noticed during the library cache build there are many entries that say no tmdb match and it shows the item as plex:itemid or local:itemid vs the usual tmdb or tvdb entry. I'm not sure why this is.

The cache builds do complete fully. Plex is in a docker container, and the pmm script is on the host, using the loopback address which is mapped to the plex port in the container. The lists do not appear to throw errors when building other than to say it's missing x shows when it's done.

The server is running ubuntu 18.04 kernel 5.4 and the latest python3 is installed. The plex instances used are both the latest production build which uses legacy plex scanners and the latest beta build as of 3/18 for the upgraded latest scanners.

I'm not sure what additional info I can provide or tests to run to help narrow the issue but I'm happy to collect or try whatever is necessary. Thank you in advance.
Any help would be greatly appreciated. 
-Ed

**Comment by meisnate12** [comment]
> I've noticed during the library cache build there are many entries that say no tmdb match and it shows the item as plex:itemid or local:itemid vs the usual tmdb or tvdb entry. I'm not sure why this is.

plex:itemid is the GUID when using the New Plex Movie Agent or New Plex TV Show Agent if these say no tmdb then the problem is most likely something to do with the meta you have for the item in plex a similar issue was raised [here](https://github.com/meisnate12/Plex-Meta-Manager/discussions/124#discussioncomment-500955) so see if that helps

local:itemid means that theres no match for this item in plex if you correct the item in plex the script should then find it.

If the item errors when being added to the cache it won't be added to any collections automatically so once you fix the problems with your items being added to the cache your other problems should work out for themselves
